<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QWidget" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_menu">
    <!-- Banner label -->
    <item>
     <widget class="QLabel" name="bannerLabel">
      <property name="text">
       <string>Welcome to the Menu</string>
      </property>
      <property name="font">
        <font>
          <pointsize>24</pointsize>
          <bold>true</bold>
        </font>
      </property>
      <property name="alignment">
        <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <!-- User Last Name label -->
    <item>
     <widget class="QLabel" name="userLastNameLabel">
      <property name="text">
       <string>Last Name: </string>
      </property>
      <property name="font">
        <font>
          <pointsize>18</pointsize>
        </font>
      </property>
     </widget>
    </item>
    <!-- Income button -->
    <item>
     <widget class="QPushButton" name="incomeButton">
      <property name="text">
       <string>Income</string>
      </property>
     </widget>
    </item>
    <!-- Expense button -->
    <item>
     <widget class="QPushButton" name="expenseButton">
      <property name="text">
       <string>Expense</string>
      </property>
     </widget>
    </item>
    <!-- Total button -->
    <item>
     <widget class="QPushButton" name="totalButton">
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
    </item>
    <!-- Log Out button -->
    <item>
     <widget class="QPushButton" name="logoutButton">
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
    </item>
   </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
